storybook/addons.ts

```diff
@@ -1,4 +1,4 @@
 import '@storybook/addon-knobs/register';
 import '@storybook/addon-actions/register';
 import '@storybook/addon-links/register';
-import './addons/DaedalusMenu/register';
+//import './addons/DaedalusMenu/register';
```

Comment:
not supported or not needed anymore?

Reply:
put back in webpack5 branch, needed some updates